qrexec/policy/parser.py

```diff
@@ -208,6 +208,20 @@ def __init__(cls, name: str, bases: Tuple[type], dict_: Dict[str, str]):
         if "PREFIX" in dict_:
             cls.prefixes[dict_["PREFIX"]] = cls # type: ignore
 
+def match_strings(info: SystemInfo, self: str, other: str) -> bool:
```

Comment:
Shouldn't this be a method of `VMToken` class?

Reply:
I believe it can be called with arguments that are not `VMToken`s, and the operation is symmetric between `self` and `other`.